src/main/scala/uk/gov/hmrc/apiplatform/modules/apis/domain/models/HttpMethod.scala

```diff
@@ -31,12 +31,9 @@ object HttpMethod {
 
   final val values = Set[HttpMethod](GET, POST, PUT, PATCH, DELETE, OPTIONS, HEAD)
 
-  def apply(text: String): Option[HttpMethod] = {
-    HttpMethod.values.find(_.toString == text.toUpperCase)
-  }
+  def apply(text: String): Option[HttpMethod] = HttpMethod.values.find(_.toString == text.toUpperCase)
 
-  def unsafeApply(text: String): HttpMethod =
-    apply(text).getOrElse(throw new RuntimeException(s"$text is not a valid API Http Method"))
+  def unsafeApply(text: String): HttpMethod = apply(text).getOrElse(throw new RuntimeException(s"$text is not a valid API Http Method"))
```

Comment:
Perhaps `is not a valid HTTP Method`.  `API` isn't in the class name, and `HTTP` is usually all caps when it's not a variable name.

Reply:
In theory they are API Http Methods because they didn't originally contain all the Http Methods